docs/application/dotnet/get-started/iot/first-app.md

```diff
@@ -0,0 +1,219 @@
+
+    The **Tizen Project Wizard** pop-up window appears.
+
+4. Select the profile, **Common** and click **OK**.
+
+    ![Tizen Project Wizard](media/cs_first_creating_project_wizard_common.png)
+
+
+The following figure illustrates a solution with the common project created and displayed in the **Solution Explorer** view:
+
+![Project Structure](media/vs_solution_explorer.png)
+
+-   The **\<projectname\>** project contains the Xamarin.Forms code shared across platforms.
+-   A common project titled **\<projectname\>.Tizen** is added. It contains code to instantiate your common application within the Tizen framework.
+
+
+The `.xaml` file in the portable project already contains simple Xamarin.Forms code that makes a basic UI.
+
+## Build Your Application
+
+After you create the application project, you can implement the required features. In this example, only the default features from the project template are used, and no code change is required.
+
+When your application code is ready, build the application. The building process performs a validation check and compiles your files. You must sign the application package with an author certificate when building the application. If you have not yet registered a Tizen certificate in Visual Studio, see [Certificate Manager](../../../vstools/tools/certificate-manager.md).
+
+There are two different ways to build the application:
+
+-   In the Visual Studio menu, select **Build \> Build Solution**.
+-   In the **Solution Explorer** view, right-click the solution name and select **Build**.
+
+Tizen .NET applications are always deployed as installable packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the \<projectname\>.Tizen project.
+
+For this example application, the default manifest is sufficient. If you want to make any change in the application, such as changing the application icon or installing resources that are used by the application at runtime, see [Package Your Application](#package-your-application).
+
+After you build the application, deploy and run it.
+
+## Deploy and Run Your Application
+
+To run the UI application, you must first deploy it to the reference board with the headed image installed. Deploying means transferring the package file (`.tpk`) to the target and invoking the Tizen Package Manager to install it.
+
+To deploy and run the application on the reference board:
+
+1. Connect the reference board to your PC.
+
+2. Once you connect the reference board, you can deploy the application by clicking the reference board instance in the Visual Studio toolbar. Make the Tizen project as a start-up project with **Set as StartUp Project**.
+
+    ![Deploy your package](media/vs_device_launch.png)
```

Comment:
Kindly move the image after line 100, like this:

2. After you connect the reference board, you can deploy the application by clicking the reference board instance in the Visual Studio toolbar. Make the Tizen project as a start-up project with **Set as StartUp Project**.

3. In the Visual Studio toolbar, you can select the reference board from the drop-down list to change the deployment target.
   ![Deploy your package](media/vs_device_launch.png)

4. If the deployment is successful, the application icon gets displayed on the device screen. Click the icon to launch the application.